db/migrate/20170812164743_add_foreign_key_craftsman_id_to_applicant.rb

```diff
@@ -0,0 +1,20 @@
+class AddForeignKeyCraftsmanIdToApplicant < ActiveRecord::Migration
```

Comment:
The migration to rm this column is dated earlier than this one so it'll cause issues if this is here.